tests/unit/ast/test_batch.py

```diff
@@ -16,16 +16,16 @@ def fake_session():
 
 
```

Comment:
can we add some tests for the transitive closure? Just in terms of what to expect? That'd be great.

Reply:
Yes, I think @sfc-gh-oplaton's recently added smoke tests are a great candidate for testing the transitive closure work.